lib/Backends/CPU/GlowJIT.cpp

```diff
@@ -137,7 +137,7 @@ GlowJIT::GlowJIT(llvm::TargetMachine &TM)
   llvm::sys::DynamicLibrary::LoadLibraryPermanently(nullptr);
 }
 
-GlowJIT::ModuleHandle GlowJIT::addModule(std::unique_ptr<Module> M) {
+GlowJIT::ModuleHandle GlowJIT::addModule(std::unique_ptr<llvm::Module> M) {
```

Comment:
why no corresponding change in the header file. Can you also add compilation error to the description please.